Update authors.yml

ashngar opened the pull request:
added author id for Ashley Nelcy Garcia

ashngar committed:
Update authors.yml

added author: Ashley Nelcy Garcia

ashngar committed:
Creating review of Más de 72 by Ashley N. García

halperta commented on _posts/blog/2018-09-27-Más-de-72.md:
```diff
@@ -0,0 +1,36 @@
+**Más de 72** <br/>
```
Above the title, please add a "YAML" header. You can find more information here:

http://www.halperta.com/criticalarchives/tutorial/writing-in-markdown/

halperta commented on _posts/blog/2018-09-27-Más-de-72.md:
```diff
@@ -0,0 +1,36 @@
+
+ 
+
+What is a digital archive? I asked myself this question weeks before submitting this review. While digital archives are typically defined as a coherent set of digital objects that have been put online by a library or archive, Más de 72 challenges the notion of what we can identify as a digital collection of records.
```
"Más de 72" should be in italics.

halperta reviewed (changes requested):
Ashley - You'll get more specific feedback from us after publication, but overall this review is very strong and almost ready for publication. 

I made a couple of comments about formatting. I'd also like to see you incorporate some kind of reference to our class readings, as the assignment states.

Please submit these changes no later than our next class period (next Thursday).

Thanks! Hannah

ashngar committed:
Update 2018-09-27-Más-de-72.md

Hi Hannah, 

I did the following: 
1. Added the YAML" header". Note, I was not able to add an image. Also, let me know if I need to reorganize the tags. 
2.  I also italicized Más de 72 
3. I referenced a class reading. 

Please let me know if I need to do anything else. 

Thank you, 
Ashley

halperta reviewed (approved):
These updates all look great. I'll do some final copyediting and we'll publish today!

halperta committed:
Final Copyedit

Corrected a few typos.

halperta committed:
Changing pub date
